filetests/isa/x86/extractlane-run.clif

```diff
@@ -0,0 +1,31 @@
+test run
+set enable_simd
+
+function %test_extractlane_b8() -> b8 {
+ebb0:
+    v0 = bconst.b8 true
+    v1 = splat.b8x16 v0
+    v2 = extractlane v1, 10
```

Comment:
Hmm, even if it reads the first lane, this test is correct. Is there a way we could use a bconst.b8x16 above?

Reply:
Yeah, I wrote a lot of this before I had worked much on vconst parsing (bda45f4ac) so unfortunately I had to use what I had. I'll change these as I see them; thanks for pointing them out.